### PR TITLE
Feature/nl header component

### LINKED ISSUE #17: Header Component Creation

### LINKED ISSUE #19: Header Component Styling

### PR DESCRIPTION
**Pull request by NicoleLatifi**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?

Create Header functional component with a log, title and login button. Add initial styling.

### Files modified:

Header.js
Header.css

### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?

Open in web browser with npm start and see the elements display with styling.

### What are the relevant tickets?

resolve #17 
resolve #19 

### Message/Questions for reviewer:

What else will the header need?

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Header.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+* {
```
Comment (relyt4me) [inline src/Header.css:1]: This might duplicate in my css addition but it should be an easy fix if so

File: src/Header.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Header.css';
+
+const Header = () => {
+  return (
+    <header className="header">
+      <img src="" alt="Bad Movie Logo" height="50" width="50" />
```
Comment (relyt4me) [inline src/Header.js:7]: We should find a fun asset for this or make one on a logo making site

Comment (NicoleLatifi) [reply src/Header.js:7]: Yes! I wanted to see if you have any ideas? Maybe something like an upside-down popcorn container? We can look around and see what we find.

File: src/Header.css (hunk 1 above)
Comment (NicoleLatifi) [reply src/Header.css:1]: Thanks for the heads up. Let's see what body css we want to use and maybe add it to App.css?

**Pull request merged**